### PR TITLE
Export Academy booking/cancellation notification

### PR DESCRIPTION
**Pull request by bsabhilash**
Gov notify for booking and cancellations

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/TICKET_ID_HERE
- [x] Jira ticket has the correct status.
- [x] [Changelog](CHANGELOG.md) entry added.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: export_academy/urls.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def registration_required(function):
         name='registration-success',
     ),
     path(
-        'booking/success/',
+        'booking/success',
```
Comment (mforner13) [inline export_academy/urls.py:35]: Not sure if this matters but all the other URLS end with a `/` - should we keep that pattern? :)

**Review by mforner13 — APPROVED** [review]
LGTM 👍 Tiny comment, but I won't block :)

**Review by rafa-garcia — APPROVED** [review]
LGTM

File: export_academy/urls.py (hunk 1 above)
Comment (bsabhilash) [reply export_academy/urls.py:35]: Ah, good point. I wonder why I changed this. Either way, the link will change in future when we tackle the booking confirmation page. This ticket was all about booking notification

**Pull request merged**